app/bundles/LeadBundle/Controller/AjaxController.php

```diff
@@ -766,11 +766,17 @@ protected function updateLeadFieldValuesAction(Request $request)
         $dataArray = ['success' => 0, 'options' => null, 'operators' => null, 'disabled' => false];
         $leadField = $this->getModel('lead.field')->getRepository()->findOneBy(['alias' => $alias]);
 
-        if ($leadField) {
+        if($alias == 'notifications'){
```

Comment:
Could you please review your code changes with Mautic CS? I can see several issues with `if(` instead of `if (` and `){` instead of `} (`.